GNUmakefile

```diff
@@ -244,6 +244,12 @@ ifeq ($(blas),mkl)
     ifeq ($(CXX),icpc)
         blas_fortran = ifort
     endif
+    # BLAS is Intel MKL and SLATE is using the SYCL backend
+    # Use ifort, threaded-blas and mkl_intel_thread
+    ifeq ($(gpu_backend),sycl)
+        blas_fortran = ifort
+        blas_threaded = 1
+    endif
```

Comment:
"Does the GNUmakefile need a similar change? Currently it has blas_threaded ?= 0 at line 60. (Which GitHub doesn't allow me to tag because it isn't near where you edited.)"
The GNUmakefile needed a similar fix.